docs/source/2019Barry.rst

```diff
@@ -120,6 +145,28 @@ Hurricane Track and Intensity
     * The time variation of maximum wind speed in GFS.v16.0.10 agrees well with Best Track.
     * The minimum sea level pressure reaches to a lower value in GFS.v16.0.10 (982 hPa) compared with Best Track (993 hPa). 
 
+  .. group-tab:: SRW.v1.0
+
+    .. figure:: images/2019Barry/tracker_Barry_srwv1.png
```

Comment:
You can change the range of the colormap scale smaller so that the color differences can be more obvious.
`norm=co.Normalize(vmin=9,vmax=55)`

Reply:
Yes, Barry is weak, we can adjust that. Thanks!